### PR TITLE
Adding first version of Phoenix User Log POC creation

### PR DESCRIPTION
**Pull request by sheyd**
#### What's this PR do?

Adds first version of POC User Log sources from Phoenix DB.

#### Where should the reviewer start?

The one script!

#### How should this be manually tested?

Already tested in Prod.

#### What are the relevant tickets?

https://github.com/DoSomething/quasar/issues/273

### CODE REVIEW

**Comment by sheyd** [comment]
@mshmsh5000 or @DFurnes Would one you mind 👀 this before pushing live?

File: etl-scripts/update_phoenix_user_log.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    db.commit()
+
+# Get List of All Phoenix Snapshots
+cur.execute("SHOW TABLES;")
+db.commit()
+
+# Iterate over tables and print name, testing
+table_snapshot = cur.fetchall()
+for x in table_snapshot:
+    table_name = str(x).strip('(').strip(')').strip(',').strip("'")
```
Comment (mshmsh5000) [inline etl-scripts/update_phoenix_user_log.py]: What's the original table name format that you're cleaning up?

**Comment by sheyd** [comment]
@mshmsh5000 Can you 👀  this PR so I can automate our User Table POC?

**Comment by mshmsh5000** [comment]
@sheyd Sorry, I had [that one question](https://github.com/DoSomething/quasar/pull/278#discussion_r92691105) for you back when you posted the PR, and never saw a response.

File: etl-scripts/update_phoenix_user_log.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          passwd=config.pw, #  password
+          db='phoenix_user_snapshots' # source and staging area for snapshots
+          )
+
+cur = db.cursor()
+
+# Record start time
+start_time = time.time()
+
+# Get List of All Phoenix Staging Tables
```
Comment (mshmsh5000) [inline etl-scripts/update_phoenix_user_log.py]: Super nit-picky, but these comments are usually just sentences—don't need to be all capitalized words.

**Comment by mshmsh5000** [comment]
The rest of this makes sense to me, just the general note on comment formatting for everyone's benefit:

```
# This Comment Doesn't Need to be All Upper Case
```

should generally be

```
# This comment doesn't need to be all upper case
```

or, if you're me,

```
# This comment doesn't need to be all upper case.
```

**Comment by sheyd** [comment]
@mshmsh5000 Updated syntax changes and answered your questions inline with comment.

**Comment by mshmsh5000** [comment]
This is python trivia, but you should be able to strip out any chars with a single command, as long as they're all at the beginning and end of the string. I think you'd get the same result from the shorter

```
table_name = str(x).strip("(),;'")
```

**Comment by mshmsh5000** [comment]
Anyhoo, 👍

**Pull request merged**